Assets/Scripts/Health.cs

```diff
@@ -8,6 +8,10 @@ public class Health : MonoBehaviour {
 	public int maxHealth = 10;
 	private int currentHealth;
 
+    public float dropProbability; // number between 0.0 and 1.0
```

Comment:
sure they can. spawning of the item only has to see if the parent is dead/alive. This can be done on a separate script (and also trigger it), and that can check if the enemy is alive or not.